javascript-if/main.js

```diff
@@ -0,0 +1,81 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if ((person.age >= 16) && (person.age >= 21)) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function categorizeAcidity(pH) {
+  if (pH < 0) {
+    return 'invalid pH level';
+  } else if (pH < 7) {
+    return 'basic';
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH <= 14) {
+    return 'acidic';
+  } else {
+    return 'invalid pH level';
+  }
+}
```

Comment:
Your acids and bases are backwards. Also, please make sure the string exactly matches the spec:

---

![image](https://user-images.githubusercontent.com/16143771/187788156-c451d295-e2d2-4a23-89bb-740e2251dd2b.png)

Reply:
@brandon-au Please address the above issue with `categorizeAcidity`